src/protocol/http/org/apache/jmeter/protocol/http/curl/BasicCurlParser.java

```diff
@@ -902,9 +919,9 @@ private void setProxyServer(Request request, String proxyServerParameters) {
     * @param request               http request
     * @param proxyServerUserPasswd the username and password of proxy server
     */
-   private void setProxyServerUserInfo(Request request, String proxyServerUserPasswd) {
-       if (proxyServerUserPasswd.contains(":")) {
-           String[] userInfo = proxyServerUserPasswd.split(":");
+   private void setProxyServerUserInfo(Request request, String authentication) {
+       if (authentication.contains(":")) {
```

Comment:
The logic seems to be the same as on line 896. Maybe we could extract it into a method?